lib/elixir/lib/option_parser.ex

```diff
@@ -353,6 +360,11 @@ defmodule OptionParser do
           f when f in [false, "false"] -> {nil, false}
           _ -> {true, value}
         end
+      :count in kinds ->
+        case value do
+          1 -> {nil, value}
+          _ -> {true, value}
+        end
```

Comment:
this section implemented new priority will be `:boolean` –> `:count` –> `:integer` –> `:float` –> `:string`.